include/arc_utilities/ros_helpers.hpp

```diff
@@ -82,6 +82,23 @@ namespace ROSHelpers
         return param_val;
     }
 
+    template <typename T>
+    inline Maybe::Maybe<T> GetParamOptional(const ros::NodeHandle& nh, const std::string& param_name, const std::string& calling_fn_name)
+    {
+        ROS_DEBUG_STREAM_NAMED("params", "No default value for " << param_name << ": Value must be on paramter sever");
+        T param_val;
+        if (nh.getParam(param_name, param_val))
+        {
+            ROS_INFO_STREAM_NAMED("params", "Retrieving " << std::left << std::setw(PARAM_NAME_WIDTH) << param_name << " as " << param_val);
+            return Maybe::Maybe<T>(param_val);
+        }
+        else
```

Comment:
`else` not needed

Reply:
This would be a sweeping change across the whole file; as written these match the style of existing functions. I'll come back and re-evaluate this style after resolving this pull request, and potentially submit a new one.